Core/Pixel.swift

```diff
@@ -168,6 +168,9 @@ public enum PixelName: String {
     
     case bookmarkImportSuccess = "m_bi_s"
     case bookmarkImportFailure = "m_bi_e"
+    case bookmarkImportFailureParsing = "m_bi_e_p"
+    case bookmarkImportFailureSaving = "m_bi_e_s"
+    case bookmarkImportFailureUnknown = "m_bi_e_u"
```

Comment:
One other thing here: [we should move to clear naming conventions where possible](https://app.asana.com/0/69071770703008/1200583483841269/f). Even though it would be inconsistent with the other import pixels, I think it makes sense to go with more verbose names here 🙂